src/main/kotlin/no/nav/familie/ef/mottak/task/SendPåminnelseOmDokumentasjonsbehovTilDittNavTask.kt

```diff
@@ -42,6 +43,14 @@ class SendPåminnelseOmDokumentasjonsbehovTilDittNavTask(
 
     private val logger = LoggerFactory.getLogger(this::class.java)
 
+    val dokumentasjonsbehovvarslinger = counter("alene.med.barn.dokumentasjonsbehovvarsling.sendt")
+    val dokumentasjonsbehovvarslingerBrukerSendtInn =
+        counter("alene.med.barn.dokumentasjonsbehovvarslingstoppet.bruker")
+    val dokumentasjonsbehovvarslingerSaksbehandlerStartet =
+        counter("alene.med.barn.dokumentasjonsbehovvarslingstoppet.saksbehandler")
+    val dokumentasjonsbehovvarslingerSaksbehandlerOgBruker =
+        counter("alene.med.barn.dokumentasjonsbehovvarslingstoppet.saksbehandlerogbruker")
+
```

Comment:
```suggestion
    val dokumentasjonsbehovVarslingerSendt = counter("alene.med.barn.dokumentasjonsbehovvarsling.sendt")
    val varslingerAvbruttGrunnetEttersending =
        counter("alene.med.barn.dokumentasjonsbehovvarslingstoppet.bruker")
    val varslingerAvbruttGrunnetPåbegyntBehandleSakOppgave =
        counter("alene.med.barn.dokumentasjonsbehovvarslingstoppet.saksbehandler")
    val varslingerAvbruttGrunnetEttersendingOgPåbegyntBehandleSakOppgave =
        counter("alene.med.barn.dokumentasjonsbehovvarslingstoppet.saksbehandlerogbruker")

```

Synes det blir lettere å forstå hva de logger med disse navnene